### ISSUE TITLE
Enhancement request- Client Contact Script

### ISSUE THREAD

**Issue by khiestand** [MN-Script-Team/DHS-MAXIS-Scripts#2773]
A few workers have asked to add checkbox for the following:
Check here if you left a generic message requesting they return call.

I think the PRISM client contact script has this in it and people like it.

**Comment by CDPotter** [comment]
Didn't it used to have that functionality? Looking at it now since I last used it as a worker it's a completely different beast. 

Personally I like the idea, since the box would be down towards the bottom it won't mess with the tabbing which was my biggest beef as a worker when people edited that script. I had the tabs memorized so it threw me off my game when new fields appeared towards the top. 

Would we want to tie actions_taken's mandatory status to this field? For example someone clicks the checkbox then that counts as their actions taken?

**Comment by IlseFerris** [comment]
I checked with Hennepin County's phone quality staff, and they like this idea as well. Would workers then also be able to bypass contact_reason and contact_type as well?

**Comment by CDPotter** [comment]
I would hesitate on loosening those mandatory fields. If I was a call center I'd want to see why you left the generic message.

**Comment by RoberT-1000** [comment]
I like adding it so long as it doesn't replace any other functionality. When the script used to have that checkbox, staff would check it and not add any other information. Call center, reception and coverage staff would have no idea what was going on with the case or the reason for the call.

**Comment by CDPotter** [comment]
Aha see I knew it used to be a button but then people couldn't use it responsibly... So in my mind the only functionality the checkbox would change would be allowing actions taken to be bypassed since the checkbox would be documenting your actions taken.

**Comment by IlseFerris** [comment]
A generic message left and information being left for the client/call center for the return call is different in my opinion.

**Comment by RoberT-1000** [comment]
Much like the "Check here to update PND2 for client delay" defaulting to checked on NOTES - CAF, with great power came great laziness...

**Comment by IlseFerris** [comment]
For instance: Left vm for client at 11:59 a.m. re: COLA income
Need to know if VA income is anticipated to change. 
---
Ilse F. 

This is not a generic message. I let the client know why I was calling and there's information for other staff of what is needed.

**Comment by RoberT-1000** [comment]
I can guarantee staff will start using the "left generic message" checkbox and skipping everything else.

**Comment by IlseFerris** [comment]
How about instead of left generic message, and requested call back, then checkbox case notes that the staff has requested a call back? I don't necessarily want staff to skip over the rest of the information, especially since we're a banking agency.

**Comment by CDPotter** [comment]
Maybe it's best left as it is? Often time I would have to edit the "left generic message for call back" text anyway, for situations where there was no VM set up, where it was someone else's VM, etc.

**Comment by MiKaylaH22** [comment]
I dont think we will use a generic message, most workers want to convey what was left. I like Ilse Idea of "requested call back"

**Comment by theVKC** [comment]
If I am remembering correctly, we had this removed because it was being over-used without clarifying remarks. It does exist in PRISM but we have not run into this situation here as far as I can tell (and if we see it, we will likely replace it as we did in MAXIS).

I'm interested to know what you-all decide since it could inform a PRISM enhancement. Personally, I like the option to add a checkbox for requesting a call back, as Ilse suggested. :)

**Comment by MiKaylaH22** [comment]
I Agree to add
